packages/useful-types/src/types.ts

```diff
@@ -53,3 +53,14 @@ export type NonOptionalKeys<T> = {
 export type NonNullableKeys<T> = {
   [K in keyof T]-?: null extends T[K] ? never : K
 }[keyof T];
+
+type ReactStatics =
+  | 'displayName'
+  | 'getDerivedStateFromProps'
+  | 'getDerivedStateFromErrors'
+  | 'childContextTypes'
+  | 'contextType'
+  | 'contextTypes'
+  | 'getDefaultProps'
+  | 'propTypes';
```

Comment:
Did you have a particular list you were referencing here?

Reply:
Add your reference `// Reference https://github.com/mridgway/hoist-non-react-statics/blob/master/src/index.js#L6`